Enable MathJax functionality

pat-s opened the pull request:
When using a custom template in combination with `self_contained` mode, MathJax does not work usually. This fact prevents using LaTeX equations in such documents.

I found a workaround [here](https://github.com/rstudio/rmarkdown/issues/727) and implemented the functionality. Also, I added a small note in `skeleton/part2.Rmd`. 

![screenshot 2017-02-20 18 01 31](https://cloud.githubusercontent.com/assets/8430564/23135124/a1af998e-f797-11e6-9834-91157b401b0d.png)

humburg commented:
Thanks, that's great. Could you please target the *develop* branch and make sure that it builds correctly? I suspect that the only thing required is to run `devtools::document`.

I think it would be better to move documentation regarding the use of MathJax to the main document since that is where the general usage instructions live. It probably should have its own subsection towards the end.

pat-s commented:
Yeah sure, forget about checking the build :no_mouth: 
The `param` I introduced in `multi_document()` was also undocumented.

I moved the instructions to the main document. Feel free to modify it if you find it to short or anything else! 

BTW: I build with `roxygen2` v6.0.1 - it automatically changed the DESCRIPTION file. In case you do not want to update your roxygen version, just ignore this commit.

humburg commented:
Thanks!
